src/SWP/Bundle/CoreBundle/Controller/WidgetController.php

```diff
@@ -84,6 +84,11 @@ public function getAction(Request $request, $id)
     /**
      * Create new widget.
      *
+     * Note:
+     *
+     *     Widget Type can be widget ID or his class (string).
+     *     Example: "SWP\\Bundle\\CoreBundle\\Widget\\ContentListWidget"
```

Comment:
Ticket is about this that you provide `integer` and get `string`. So this fix that without breaking BC. Short names for WidgetsHandler is something else and should be discussed. I see issue with storing class as type name (as we do this now) as we will need to run migration if we change class namespace or name. This is fast fix. You talk about change in concept.